### PR TITLE
ElNotification: define types for `ElNotification` close and closeAll.

### PR DESCRIPTION
**Pull request by underfin**
Please make sure these boxes are checked before submitting your PR, thank you!

* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md) | [Español](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.es.md) | [Français](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.fr-FR.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

### CODE REVIEW

File: types/notification.d.ts [hunk 1]
```diff
@@ -74,6 +74,12 @@ export interface ElNotification {
 
   /** Show an error notification */
   error (options: ElNotificationOptions): ElNotificationComponent
+
+  /** Close the notification, if not define id, will close first notification */
+  close(id?: string, onClose?: Function): void
```
Comment (luckyCao) [inline types/notification.d.ts]: close 的方法第一个参数不是可选的

Comment (underfin) [reply types/notification.d.ts]: 没有传的话，默认删除第一个
```
const instance = instances.filter((instance, i) => {
    if (instance.id === id) {
      index = i;
      return true;
    }
    return false;
  })[0];
```

**Comment by underfin** [comment]
I already change code.Thanks for your review.

**Comment by ty793941** [comment]
Wait，I still Can't see **closeAll** function in notification.d.ts at dev branch. Do I have something wrong ?

link:https://github.com/ElemeFE/element/blob/dev/types/notification.d.ts
image

**Comment by luanweijie** [comment]
declare module 'element-ui/types/notification' {
  interface ElNotification {
      /** Close the notification by id */
    close(id: string, onClose?: Function): void
    /** Close all notification */
    closeAll(): void
  }
}

**Pull request closed without merging**